src/test/kotlin/com/hypto/iam/server/apis/UserApiTest.kt

```diff
@@ -339,6 +341,90 @@ class UserApiTest : AbstractContainerBaseTest() {
         }
     }
 
+    @Test
+    fun `change root user password - success`() {
```

Comment:
Add cases for 
1. wrong old password
2. userA calling this API to change userB's password without "changePassword" permission